python/llm/scripts/env-check.sh

```diff
@@ -125,6 +132,42 @@ check_ipex()
   fi
 }
 
+check_xpu_info()
+{
+  echo "-----------------------------------------------------------------"
+  lspci -v | grep -i vga -A 8
+}
+
+check_linux_kernel_version()
+{
+  echo "-----------------------------------------------------------------"
+  uname -a
+}
+
+check_xpu_driver()
+{
+  echo "-----------------------------------------------------------------"
+  xpu-smi -v
+}
+
+check_OpenCL_driver()
```

Comment:
Comment this out since it has sudo as well?

Reply:
I tried it again. It didn't need `sudo`.